### DISCUSSION TITLE
This blob took too long to generate

### DISCUSSION THREAD

**Discussion by br14** [community/community#66045] [Repositories] [answered]
### Select Topic Area

Question

### Body

Looking at Node release data, and getting the message:

"This blob took too long to generate"

https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md#18.17.1

Am I doing something wrong, or has Github reduced the amount of time allocated for generating the markdown file? Fairly sure it was working a few weeks ago.

![image](https://github.com/community/community/assets/418641/b8d102d6-9efc-4f20-a3ee-515e56f2d036)

**Comment by lab596** [comment, marked as answer]
It's unlikely that you are doing something wrong in this case. It's more likely that the file you are trying to access is particularly large or that GitHub's servers are experiencing high traffic or load at that moment.

Here's what you can do:

1. View in Raw Mode: If the file isn't too large, you can try viewing it in "Raw" mode by appending `?raw=true` to the URL. For example: https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md?raw=true (this link works for me to see the blob).
2. Wait and Retry: Sometimes, this issue is temporary, and waiting a while and then retrying can resolve it.
